README.md

```diff
@@ -1 +1,39 @@
-# space-hub
\ No newline at end of file
+# Space Hub
+@Gopxfs and @worashf built a Space Hub application, allowing users to check on SpaceX rockets and missions (fetching data from SpaceX's API). A profile section was also implemented, displaying the missions and rockets currently booked. React & Redux were used to create and dispatch actions, and React Testing Library & Jest were used to test the components/functions. The styling was made using React Bootstrap and plain CSS3 files.
+
+## Built With:
+- React & Redux
+- React Bootstrap
+- JavaScript
+
+## Live demo:
+Not ready for deployment.
+
+## Getting Started:
+To access and change the code on your computer (using GIT):
+> Note: if you don't use git, you can download the ZIP of this repository by clicking on the "Code" button above the files list.
+### Setup
+- Open the repository [link](https://github.com/Gopxfs/space-hub).
+- Click on the "Code" button, above the files list.
+- Select one of the three options to copy the URL: HTTPS, SSH key and GitHub CLI.
+- Open the Terminal in the directory you want to copy the project.
+- Type git clone, paste the URL you copied, and press enter.
+### Usage
+To edit this project:
+- Open the repository you cloned with a code editor that you like.
+- Edit the code as you want.
+
+## Authors:
+:bust_in_silhouette: Gabriel Fonseca
+- Github: [@Gopxfs](https://github.com/Gopxfs)
+- Linkedin: [Gabriel Fonseca](https://www.linkedin.com/in/gabriel-fonseca-sales-8bb64b236/)
+
+:bust_in_silhouette: Worash Bocherugn
+- Github: [@worashf](https://github.com/worashf)
+- Linkedin: [Worash Bocherugn](https://www.linkedin.com/in/worash-abocherugn-a02219154/)
+
+## :star2: Show your support
+Give a :star: if you like my project!
+
+## :pencil: License
+This project is [MIT](https://github.com/Gopxfs/space-hub/blob/main/LICENSE) licensed.
```

Comment:
**Optional**
- Please fix the MIT link for a professional README file

Reply:
Thanks for the review! We are going to merge it 😄